tests/test_core.py

```diff
@@ -144,4 +144,36 @@ def test_series2cat():
     series2cat(df,*cols)
     for col in cols:
         assert (df[col].dtypes == 'category')
-    assert (df['col3'].dtypes == 'int64')
\ No newline at end of file
+    assert (df['col3'].dtypes == 'int64')
+
+def _write_file(name):
+    with open(name, 'w') as f: f.write(str(name))
+def _clean_up(fns):
+    for fn in fns: os.remove(fn)
+
+class TestMaybeCopy(object):
+    @pytest.mark.parametrize('old_fnames, new_fnames', [
+        (['./first'], ['./second']),
+        ([Path('./first')], [Path('./second')]),
+    ])
+    def test_copies_if_does_not_exist(self, old_fnames, new_fnames):
+        for fn in old_fnames: _write_file(fn)
+        maybe_copy(old_fnames, new_fnames)
+        for fn in new_fnames: assert os.path.exists(fn)
+        _clean_up(old_fnames+new_fnames)
+
+    def test_copies_if_older(self):
+        _write_file('first')
```

Comment:
Probably best to use the `tempfile` std lib and a `with` block to ensure that the files are created somewhere appropriate, and automatically cleaned up when done.